pkg/gomod/parse.go

```diff
@@ -0,0 +1,36 @@
+package gomod
+
+import (
+	"bufio"
+	"io"
+	"strings"
+
+	"github.com/aquasecurity/go-dep-parser/pkg/types"
+)
+
+// Parse parses a go.sum file
+func Parse(r io.Reader) ([]types.Library, error) {
+	var libs []types.Library
+	tmp := make(map[string]string)
+
+	scanner := bufio.NewScanner(r)
+	for scanner.Scan() {
+		line := scanner.Text()
+		s := strings.Fields(line)
+		// go.sum records and sorts all non-major versions
+		// with the latest version as last entry
+		tmp[s[0]] = strings.TrimSuffix(strings.TrimPrefix(s[1], "v"), "/go.mod")
+	}
+	if err := scanner.Err(); err != nil {
+		return nil, err
```

Comment:
```suggestion
		return nil, xerrors.Errorf("scan error: %w", err)
```
Could you wrap the error?

Reply:
done, wrap errors in other pkgs as well

Reply:
Thanks!